### PR TITLE
Introduce numeric attributes

### PR DESCRIPTION
**Pull request by IKarbowiak**
Allow creating and assigning numeric attributes.
[Dasboard PR](https://github.com/mirumee/saleor-dashboard/pull/1065)

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [x] The changes are tested
* [x] GraphQL schema and type definitions are up to date
* [x] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by trevorlelsbree** [comment]
When will this feature be completed? Or released

**Review by maarcingebala — APPROVED** [review]

File: saleor/graphql/schema.graphql [hunk 1]
```diff
@@ -380,6 +380,15 @@ type AppUpdate {
   app: App
 }
 
+enum AreaUnitsEnum {
```
[the remark this answers is not in the document]
Comment (d-wysocki) [reply saleor/graphql/schema.graphql:383]: @IKarbowiak we can remove `Enum` suffix from `AreaUnitsEnum `, `MeasurementUnitsEnum ` and `VolumeUnitsEnum ` if there is no conflicts with existing `Type's`, for example if we have `AreaUnits` Type, we should stay with `Enum` suffix.

[the remark this answers is not in the document]
Comment (IKarbowiak) [reply saleor/graphql/schema.graphql:383]: I'm waiting for #6822 to be merged.

[the remark this answers is not in the document]
Comment (maarcingebala) [reply saleor/graphql/schema.graphql:383]: #6822 is waiting for frontend and is currently blocked, I'm afraid we need to wait maybe even two weeks :/

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**